cmd/storagenode-updater/windows.go

```diff
@@ -32,6 +33,8 @@ func init() {
 	if err != nil {
 		panic("Service failed: " + err.Error())
 	}
+	// avoid starting main() when service was stopped
+	os.Exit(0)
```

Comment:
I don't quite follow the comment. Can you elaborate?

Reply:
Workflow for service is different than starting from cmd. When app is controlled by Windows whole management is done in `windows.go` file and in `init()` we have `err = svc.Run("storagenode-updater", &service{})` which blocks control until service is not stopped. Unfortunately, side effect is that when service is stopped control is returned to `init()` and after that `main()` method is executed where an application is started once again. Fix for that is exit application just after service return control and avoid going to `main()`.